tests/common/reboot.py

```diff
@@ -277,6 +281,12 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
         if not ret:
             raise Exception('warmboot-finalizer service timeout on DUT {}'.format(hostname))
 
+    # Verify if the temporary file created in tmpfs is deleted after reboot, to determine a
+    # successful reboot
+    file_check = duthost.command('find /dev/shm/test_reboot', module_ignore_errors=True)['rc']
```

Comment:
Why use ` module_ignore_errors=True`? You do care about the errors in this call, don't you?

Reply:
yes, in successful case this command with return error only which we need. If we do not use module_ignore_errors, ansible does not save the error output.